src/createStripe.ts

```diff
@@ -1,16 +1,17 @@
-import * as _Error from './Error';
```

Comment:
I renamed this file because stripe.common.ts compiles to stripe.common.js, which I think is confusing in the context of a dual package

Reply:
Not a fan of the `createStripe` name tbh :) `stripe.core`? `stripeClient`? `stripeFactory`? Not a strong opinion.

Reply:
I'm a fan of stripe.core